packages/web/app/src/components/layouts/target.tsx

```diff
@@ -366,144 +366,291 @@ export function ConnectSchemaModal(props: {
     )?.node;
   }, [selectedGraph]);
 
+  const target = query.data?.target;
+
+  const [selectedArtifact, setSelectedArtifact] = useResetState<CdnArtifactType>(
```

Comment:
We want to make sure that supergraph is selected by default for federation projects as this is what they usually want to use.